### PR TITLE
Remove unused templates

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Read below
| Type?             | refacto
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Auto tests are sufficient.
| UI Tests          | https://github.com/Hlavtox/ga.tests.ui.pr/actions/runs/7357479791
| Fixed issue or discussion?     | 
| Related PRs       | 
| Sponsor company   | 

### Description
- Made a tool that will list all twig templates in the source code and check if the template is used at least once. [usageChecker.zip](https://github.com/PrestaShop/PrestaShop/files/13794205/usageChecker.zip)
- After manual check, the result is about 15 templates that are unused.
- I also changed the paths to some templates, so it's unified with other paths.

### Unused templates
'src/PrestaShopBundle/Resources/views/Admin/Improve/Design/ThemesCatalogPage/addons_store.html.twig',
'src/PrestaShopBundle/Resources/views/Admin/Improve/Module/addons_store.html.twig',
'src/PrestaShopBundle/Resources/views/Admin/Module/Includes/card_grid_addons.html.twig',
'src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/keyword.html.twig',
'src/PrestaShopBundle/Resources/views/Admin/Common/pagination_quicknav.html.twig',
'src/PrestaShopBundle/Resources/views/Admin/Common/unavailable-feature.html.twig',
'src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories_catalog.html.twig',
'src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/delete_block.html.twig',
'src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/Modal/add_product_modal.html.twig',
'src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/Modal/update_product_modal.html.twig',
'src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/form_table_layout.html.twig',
'src/PrestaShopBundle/Resources/views/Admin/Exception/not_found.html.twig',
'src/PrestaShopBundle/Resources/views/Admin/Helpers/dropdown_menu.html.twig',
'src/PrestaShopBundle/Resources/views/Admin/Helpers/range_inputs.html.twig',
'src/PrestaShopBundle/Resources/views/Admin/Helpers/range_slider.html.twig',

### CODE REVIEW

**Review by ShaiMagal — APPROVED** [review]

**Review by FabienPapet — APPROVED** [review]

**Review by matks — CHANGES_REQUESTED** [review]
Even if these templates are not used in the software, it's a BC break 😉 

I would suggest checking git history for each of these files to make sure they are not being included using dynamic template naming (= we compute the name of the template at runtime)

File: src/PrestaShopBundle/Resources/views/Admin/Helpers/dropdown_menu.html.twig
Comment (kpodemski) [inline src/PrestaShopBundle/Resources/views/Admin/Helpers/dropdown_menu.html.twig:1]: @Hlavtox what's the point of removing something that is treated as a helper? If it's a helper, it means that there's a chance somebody from the ecosystem uses it. Removing it for nothing is not worth it

Comment (Hlavtox) [reply src/PrestaShopBundle/Resources/views/Admin/Helpers/dropdown_menu.html.twig:1]: That's what reviews are for. :-) Tell me what files you think are good to keep and I will remove them from the PR.

**Comment by Hlavtox** [comment]
@matks

> I would suggest checking git history for each of these files to make sure they are not being included using dynamic template naming (= we compute the name of the template at runtime)

Yes, that is the case for unused templates in these paths.
- Common/Grid/Blocks/EmptyState
- Common/Grid/Actions
- Common/Grid/Columns

**Review by M0rgan01 — APPROVED** [review]

**Pull request closed without merging**